src/index.ts

```diff
@@ -41,6 +24,23 @@ startupMessage += "---".repeat(30);
  *
  */
 async function main() {
+	const botConfig = await setBotConfig(process.env);
+
+	let startupMessage = "===".repeat(30);
+	startupMessage += "\n**White Whale Bot**\n";
+	startupMessage += "===".repeat(30);
+	startupMessage += `\nEnvironment Variables:\n
+**RPC ENPDOINT:** \t${botConfig.rpcUrls}
```

Comment:
you need to add logging regarding the setup of the RPCS, how many were entered, how many fetched from chain registry? whats the final list? Same for the ignore addresses. Make sure to log it is used and what the result is.